.github/workflows/release.yaml

```diff
@@ -0,0 +1,18 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
+
+permissions:
+  contents: write
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Release
+        uses: softprops/action-gh-release@v2
```

Comment:
We use the same action in github-actions repo.
https://github.com/tx-pts-dai/github-workflows/releases/tag/v0.10.0

I would create another workflow for tagging but we still need to discuss how to handle versions programatically.  

I'm not sure we should rely on using the UI for anything since we'll have inconsisstant releases.

Reply:
@swibrow Ok I see, it's creating the Changelog by itself. So for me it's good. 

> I would create another workflow for tagging but we still need to discuss how to handle versions programatically.

Ok, let's bring this up soon.